platform/android/java/lib/src/org/godotengine/godot/utils/GodotNetUtils.java

```diff
@@ -79,4 +85,29 @@ public void multicastLockRelease() {
 			Log.e("Godot", "Exception during multicast lock release: " + e);
 		}
 	}
+
+	static public String getCACertificates() {
+		try {
+			KeyStore ks = KeyStore.getInstance("AndroidCAStore");
+			StringWriter writer = new StringWriter();
```

Comment:
You want to use `StringBuilder` instead of `StringWriter` (e.g: https://stackoverflow.com/a/2980852)